### PR TITLE
Remove unnecessary computed fields

### PR DESCRIPTION
**Pull request by AlanCoding**
Objective: housekeeping so that within this dev cycle we make modest progress to removing computed fields

We had some cases where a boolean field was just referencing an integer field. Computing those as a part of the computed fields task was actually imposing addition queries. While we could optimize that away, better to stop saving trivial fields altogether. For the few that we can remove, I'm removing here.

Next item, the field `groups_with_active_failures` is not used in _any_ javascript code. Therefore, we should put users on notice of its future removal. It is possibly the most expensive field to compute, and apparently has no measurable value.

Example of new OPTIONS

```json
            "total_groups": {
                "type": "integer",
                "label": "Total groups",
                "help_text": "Total number of groups in this inventory.",
                "min_value": 0
            },
            "groups_with_active_failures": {
                "type": "integer",
                "label": "Groups with active failures",
                "help_text": "DEPRECATED: Number of groups in this inventory with active failures. This field will be removed in a future release.",
                "min_value": 0
            },
            "has_inventory_sources": {
                "type": "boolean",
                "label": "Has inventory sources",
                "help_text": "Flag indicating whether this inventory has any external inventory sources."
            },
```

I'm still running through an inventory building test, but otherwise the API testing shows that this is doing fine.

### CODE REVIEW

File: awx/main/models/inventory.py [hunk 1]
```diff
@@ -176,6 +167,14 @@ def get_absolute_url(self, request=None):
 
     variables_dict = VarsDictProperty('variables')
 
+    @property
+    def has_active_failures(self):
```
Comment (chrismeyersfsu) [inline awx/main/models/inventory.py:171]: Makes it not queriable :(

File: awx/main/models/inventory.py [hunk 2]
```diff
@@ -176,6 +167,14 @@ def get_absolute_url(self, request=None):
...
+        return bool(self.hosts_with_active_failures)
+
+    @property
+    def has_inventory_sources(self):
```
Comment (chrismeyersfsu) [inline awx/main/models/inventory.py:175]: same with the queries.

File: awx/main/models/inventory.py (hunk 1 above)
Comment (AlanCoding) [reply awx/main/models/inventory.py:171]: `/api/v2/inventories/?hosts_with_active_failures__gt=0`

**Comment by AlanCoding** [comment]
Okay, the computed fields task seems to be working fine when creating/modifying inventories through the API.

**Comment by matburt** [comment]
Reopen PR with deprecation notes but steer clear of changes that would affect `filters`

**Comment by AlanCoding** [comment]
We have an issue (will dig up) to remove computed fields wholesale.

TODO: pull back in the field deprecation warning to keep that on the agenda

**Pull request closed without merging**